pkg/cluster/version.go

```diff
@@ -40,34 +58,6 @@ func (service *openShiftClusterDocumentVersionerService) Get(ctx context.Context
 		}
 	}
 
-	errUnsupportedVersion := api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, "properties.clusterProfile.version", "The requested OpenShift version '%s' is not supported.", requestedInstallVersion)
-
-	// when we have no OpenShiftVersion entries in CosmoDB, default to building one using the InstallStream
-	if len(activeOpenShiftVersions) == 0 {
-		if requestedInstallVersion != version.InstallStream.Version.String() {
-			return nil, errUnsupportedVersion
-		}
-
-		installerPullSpec := env.LiveConfig().DefaultInstallerPullSpecOverride(ctx)
```

Comment:
Noting that in this change we're losing the "LiveConfig" capability to override the default installer pullspec. We aren't using this feature currently, so its not breaking anything, but worth noting that we may want to reintroduce this when LiveConfig is put live.